framework/src/utils/RankTwoTensor.C

```diff
@@ -211,6 +211,38 @@ RankTwoTensor::fillFromInputVector(const std::vector<Real> & input, FillMethod f
   }
 }
 
+void
+RankTwoTensor::fillFromScalarVariable(const VariableValue & scalar_variable)
+{
+  switch (scalar_variable.size())
+  {
+    case 1:
+      zero();
+      (*this)(0, 0) = scalar_variable[0]; // S11
+      break;
+
+    case 3:
+      zero();
+      (*this)(0, 0) = scalar_variable[0]; // S11
+      (*this)(1, 1) = scalar_variable[1]; // S22
+      (*this)(0, 1) = scalar_variable[2]; // S12
```

Comment:
Wait a minute... why is this one upper diagonal but the three dimensional case is written symmetric?

Reply:
This one should be symmetric too.